src/components/ContentHighlights/HighlightStepper/ContentHighlightStepper.jsx

```diff
@@ -0,0 +1,110 @@
+import React, { useState, useReducer, useEffect } from 'react';
+import {
+  Stepper, FullscreenModal, Container, Button,
+} from '@edx/paragon';
+import PropTypes from 'prop-types';
+import {
+  setHighlightStepperModal,
+} from '../data/actions';
+import { initialStepperModalState, stepperModalReducer } from '../data/reducer';
+import HighlightStepperTitle from './HighlightStepperTitle';
+import HighlightStepperSelectCourses from './HighlightStepperSelectCourses';
+import HighlightStepperConfirmCourses from './HighlightStepperConfirmCourses';
+import HighlightStepperConfirmHighlight from './HighlightStepperConfirmHighlight';
+import HighlightStepperFooterHelpLink from './HighlightStepperFooterHelpLink';
+
+const ContentHighlightStepper = ({ openModal }) => {
+  /* Stepper Modal currently accessible from:
+  - ContentHighlightSetCard.jsx
+  - CurrentContentHighlightHeader.jsx
+  - ZeroStateHighlights.jsx
+  */
```

Comment:
nit: nice comment 😄 suggestion to move this to a JSDoc above the function definition

```jsx
/**
 *. Stepper Modal currently accessible from:
 *   - ContentHighlightSetCard
 *   - CurrentContentHighlightHeader
 *   - ZeroStateHighlights
 *
 * @param {object} args Arguments
 * @param {bool} args.openModal Whether the modal containing the stepper is currently open.
 */
const ContentHighlightStepper = ({ openModal }) => {
  // ...
}
```